src/utils/requests.tsx

```diff
@@ -4,30 +4,41 @@ import { API_URL } from 'config'
 
 export const getPages = async (mangafile: string, chapter: string): Promise<Page[]> => {
 
-    const url = `${API_URL}/${mangafile}/jpg/${chapter}`;
-    let {data: pages} = await axios.get(url);
-
-    // filter out useless stuff
-    pages = pages.map((page: Record<string, unknown>) => (
-      {
-        name: page.name,
-      }
-    ));
-    return pages;
+  const url = `${API_URL}/${mangafile}/jpg/${chapter}`;
+  let { data: pages } = await axios.get(url);
+
+  // filter out useless stuff
+  pages = pages.map((page: Record<string, unknown>) => (
+    {
+      name: page.name,
+    }
+  ));
+  return pages;
 };
 
 export const getChapters = async (mangafile: string): Promise<Chapter[]> => {
 
-    const url = `${API_URL}/${mangafile}/jpg`;
-    let {data: chapters} = await axios.get(url);
-
-    // filter out useless stuff
-    chapters = chapters.reduce((result: Chapter[], chapter: Record<string,unknown>) => {
-      const name = chapter.name as string;
-      if (!name.includes('.zip')) {
-        result.push({name: name})
-      }
-      return result;
-    }, []);
-    return chapters;
+  const url = `${API_URL}/${mangafile}/jpg`;
+  let { data: chapters } = await axios.get(url);
+
+  // filter out useless stuff
+  chapters = chapters.reduce((result: Chapter[], chapter: Record<string, unknown>) => {
+    const name = chapter.name as string;
```

Comment:
can chapter.name be null for any reason?

Reply:
no because that would mean you have no file name (i think)